File: 2016/gara2/combinazione/att/combinazione.pas
function premi(N: longint; var B: array of longint): int64;
begin
    (* Mettete qui il codice della soluzione *)
    premi := 42;
end;


const
    MAXN = 100000;
var
    N, i  : longint;
    B     : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, B[i]);

    writeln(fw, premi(N, B));
    close(fr);
    close(fw);
end.

File: ed2/gara2/combinazione/att/combinazione.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

long long premi(int N, int B[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int B[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &B[i]));

    fprintf(fw, "%lld\n", premi(N, B));
    fclose(fr);
    fclose(fw);
    return 0;
}
